### PR TITLE
March Mapping Madness - New space ruin

### PR DESCRIPTION
**Pull request by Astrogem2**
I have made a new space ruin which is based off of the game Prey, cause mimics are cool, and I think some variety in space ruins would be swell
## About The Pull Request
This pull request is about a new ruin I have made, which I have taken inspiration on by the hit 2017 game Prey, which I enjoy.

prey 2, the peryer

### Mapping March
Ckey to receive rewards: Astrogem2

## Why It's Good For The Game
We don't have that many Ruins out in space, and some new ones would be nice to see.

## Changelog
:cl: Astrogem2

add: New space ruin that I have made, which adds some variety to space searching.

/:cl:

### CODE REVIEW

**Comment by VladinXXV** [comment]
Can you include pictures of the ruin in the PR body please? You might also want to expand your changelog entry.

**Comment by Astrogem2** [comment]
ight

**Comment by san7890** [comment]
that asteroid is way too big for so much regular rock, trim down the overall map size or just splinter that big asteroid into a bunch of smaller ones

**Comment by Astrogem2** [comment]
got it

**Comment by san7890** [comment]
be sure to edit your PR body with the latest version of the map to ensure that newer people who come in and see this PR don't give feedback on an outdated version of the map, thanks

**Comment by wesoda25** [comment]
Love the little simple ones like this 👍

**Comment by Addust** [comment]
plating under the windows
hivelords, holy shit rare

**Comment by Addust** [comment]
also it needs a ruin config thingymajig (check other ruin PRs!)

**Comment by Valorium-RB** [comment]
The foam force crossbow should be renamed to the Huntress Boltcaster, to complete the reference.

**Comment by Astrogem2** [comment]
damn i’m gaming right now, i’ll do these after

**Comment by Astrogem2** [comment]
ok i've done that now i'm pretty sure.

**Comment by san7890** [comment]
they aren't in your PR. push your commit to remote, or commit your staged changes and then push to remote.

**Comment by Astrogem2** [comment]
uhhhhh, ill try to do that i guess.

**Comment by Astrogem2** [comment]
wait i think i made another pr for that by mistake

**Comment by Astrogem2** [comment]
ok this time its sure to work im pretty sure, i hope

**Comment by Astrogem2** [comment]
oh christ

**Comment by Astrogem2** [comment]
this might fix the one linter error.

**Comment by Astrogem2** [comment]
ok im praying that this works now.

**Comment by san7890** [comment]
fix your conflicts, and also add a `/obj/effect/baseturf_helper/asteroid/airless` into the ruin's area (maybe making a new area for it as well, it's not that hard) such that if anyone decides to use a grenade, it'll only scrape away the walls of the ruin, and not punch a hole into space.

File: code/datums/ruins/space.dm [hunk 1]
```diff
@@ -356,3 +355,9 @@
 	suffix = "the_faceoff.dmm"
 	name = "The Faceoff"
 	description = "What do you get when a meeting of the enemy corporations get crashed?"
+
+/datum/map_template/ruin/space/prey_pod
+	id = "prey"
+	suffix = "prey_pod"
```
Comment (Maurukas) [inline code/datums/ruins/space.dm:361]:
```suggestion
	suffix = "prey_pod.dmm"
```

This needs to be the filename

**Review by Maurukas — CHANGES_REQUESTED** [review]
Looks good. Please fix the file name.

If you'd like, you can give the terminal custom text. Look at `code/modules/mapfluff/ruins/spaceruin_code/dangerous_research.dm` and `TheDerelict.dm` for examples.

File: config/spaceruinblacklist.txt [hunk 2]
```diff
@@ -54,5 +54,6 @@
 #_maps/RandomRuins/SpaceRuins/hellfactory.dmm
 #_maps/RandomRuins/SpaceRuins/space_billboard.dmm
 #_maps/RandomRuins/SpaceRuins/spinwardsmoothies.dmm
-#_maps/RandomRuins/SpaceRuins/dangerous_research.dmm
+#_maps/RandomRuins/SpaceRuins/dangerous_research.dm
```
Comment (Maurukas) [inline config/spaceruinblacklist.txt:57]:
```suggestion
#_maps/RandomRuins/SpaceRuins/dangerous_research.dmm
```

**Review by Maurukas — APPROVED** [review]

**Pull request merged**